client/my-sites/domains/domain-management/edit/domain-types/registered-domain-type.jsx

```diff
@@ -289,7 +292,11 @@ class RegisteredDomainType extends React.Component {
 		const { domain, selectedSite, purchase, isLoadingPurchase } = this.props;
 		const { name: domain_name } = domain;
 
-		const { statusText, statusClass, icon } = resolveDomainStatus( domain, purchase );
+		const { statusText, statusClass, icon } = resolveDomainStatus(
+			domain,
+			purchase,
+			pick( this.props, 'isDomainOnlySite' )
```

Comment:
Tiny but personally I prefer to avoid using lodash if I can. It's up to you.

Reply:
I share the same sentiment.